packages/util/src/NumericRepresentation.ts

```diff
@@ -1,14 +1,21 @@
 import { stringify } from "uuid";
 
+/**
+ * Converts a byte array to a hex string.  Opposite of fromHexString().
+ */
 export const toHexString = (bytes: Uint8Array) =>
```

Comment:
something i'm enforcing in `passport-server` (via eslint https://github.com/proofcarryingdata/zupass/blob/main/apps/passport-server/.eslintrc.js#L22-L28) that i haven't gotten around to enforcing in the rest of the packages is explicit return types for all functions. this reminds me that I should get around to that. https://github.com/proofcarryingdata/zupass/issues/553

Reply:
Another medium-strength preference I have is to define functions that are assigned to variables using

```ts
function x(...) {
  ...
}
```

syntax rather than arrow function syntax

Reply:
I was wondering why these were defined in this style in Brian's code.  I can switch it.

Reply:
also, fyi, the only time arrow functions make a practical difference is that they properly capture the value of `this` keyword properly, whereas the `function` syntax does not. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this